lib/jekyll_pages_api/page.rb

```diff
@@ -3,14 +3,17 @@
 module JekyllPagesApi
   # wrapper for a Jekyll::Page
   class Page
+    HTML_EXTS = %w(.html .md .markdown .textile)
     attr_reader :page
 
-    def initialize(page)
+    def initialize(page, site)
       @page = page
+      @site = site
     end
 
     def html?
-      %w(.html .md).include?(self.page.ext)
+      path = @page.path
```

Comment:
Although, I guess it also enables one invoke a method on the same object should you decide to switch from a bare instance var to an accessor... OK, I've overruled my own objection. :-P